player.js

```diff
@@ -0,0 +1,66 @@
+var Scrabble = require("./scrabble.js");
+
+var Player = function(name) {
+  this.name = name;
+  this.played_words = [];
+  this.score = 0
+}
+
+Player.prototype.plays = function () {
+  return this.played_words;
+}
+
+Player.prototype.play = function (word) {
+  if (this.hasWon() === true) {
+    return false
+  }
+  if (typeof word === "object") {
+    for (var i = 0; i < word.length; i++) {
+      this.played_words.push(word[i]);
+    }
+  }
+  else {
+    this.played_words.push(word);
+  }
+  this.total_score()
+  return this.plays();
+}
+
+Player.prototype.total_score = function () {
+  var scrabble = new Scrabble();
+  var sum = 0
+  for (var i = 0; i < this.played_words.length; i++) {
+    sum += scrabble.score(this.played_words[i]);
+  }
+  this.score = sum;
+  return this.score;
+}
+
```

Comment:
A lot of these if/else statements can be rewritten to use ternary operators, so the following code would look like this:
if this.score >= 100 ? true : false;